subsys/retention/Kconfig

```diff
@@ -20,13 +20,22 @@ config RETENTION_INIT_PRIORITY
 	  priorities for retained memory drivers.
 
 config RETENTION_MUTEXES
-	bool "Retention mutex support"
+	bool
 	default y
 	depends on MULTITHREADING
+	depends on !RETENTION_MUTEX_FORCE_DISABLE
+
+config RETENTION_MUTEX_FORCE_DISABLE
+	bool "Disable retention mutex support"
+	depends on MULTITHREADING
 	help
-	  Use mutexes to prevent issues with concurrent retention device
-	  access. Should only be disabled whereby retained memory access is
-	  required in an ISR or for special use cases.
+	  Disable use of mutexes which prevent issues with concurrent retention
+	  device access. This option should should only be enabled when
```

Comment:
s/should should/should/

Reply:
Good catch, have updated. I've added `no-mutex` support to retained memory drivers too but was a bit hesitant as it's not really a "driver" feature but seems like it's worth adding as someone might want multiple areas and requiring mutex support to be disabled for all of them is a bit of a nuisance